### PR TITLE
[ecm] update to 5.110.0

### PR DESCRIPTION
**Pull request by jiayuehua**
- [X] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md)
- [X] SHA512s are updated for each updated download
- [ ] ~The "supports" clause reflects platforms that may be fixed by this new version~
- [ ] ~Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.~
- [ ] ~Any patches that are no longer applied are deleted from the port's directory.~
- [X] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [X] Only one version is added to each modified port's versions file.

### CODE REVIEW

File: ports/ecm/portfile.cmake [hunk 1]
```diff
@@ -1,8 +1,8 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO KDE/extra-cmake-modules
-    REF v5.98.0
-    SHA512 355bf2010dd4c736981d0f708ab7245c0925104e19da01acee0482494f815088e5d7aed9aa968251f13c2c5b8cbfd67dc8bb147dec46050fea0881c00f456d9d
+    REF "v${VERSION}"
+    SHA512 fc0c6e323fcc8862c4e093ced9571647108708fbdab474ef34dbc751ac5373d9b107bec53e56e09ede62dc56b36dad096d55f08f03f1f29f634219347df7d06f
     HEAD_REF master
     PATCHES
         fix_canberra.patch         # https://invent.kde.org/frameworks/extra-cmake-modules/-/merge_requests/187
```
Comment (BillyONeal) [inline ports/ecm/portfile.cmake:8]: Are these patches still necessary?

**Review by BillyONeal — CHANGES_REQUESTED** [review]

File: ports/ecm/portfile.cmake (hunk 1 above)
Comment (jiayuehua) [reply ports/ecm/portfile.cmake]: not neccessary any more, should delete them.

**Comment by Cheney-W** [comment]
/mnt/vcpkg-ci/buildtrees/kf5solid/install-x64-linux-dbg-out.log
https://dev.azure.com/vcpkg/_apis/resources/Containers/24564343/failure%20logs%20for%20x64-linux?itemPath=failure%20logs%20for%20x64-linux%2Fkf5solid&%24format=zip&saveAbsolutePath=false

**Comment by Cheney-W** [comment]
Moving the PR to draft. Please set it to "ready for review" once failures have been fixed.

**Comment by jiayuehua** [comment]
I don't know why k5solid fail.

**Pull request closed without merging**